chat/src/main/resources/application.yml

```
@@ -19,7 +19,7 @@ spring:
 
     jpa:
       hibernate:
-        ddl-auto: update
+        ddl-auto: create
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)